octoprint_mrbeam/camera/undistort.py

```diff
@@ -148,6 +145,8 @@ def save_debug_img(img, name):
 
 	if debug_out or undistorted:
 		save_debug_img(img, "undistorted")
+	if analytics_callback:
+		analytics_callback(img)
```

Comment:
I think it is best to not add the analytics here. I strived to keep the analytics to lidhandler.py.

+ This call is blocking. You probably want to either have it run after the client has been served (i.e. doing it from lidhandler.py) or run it in parallel.
+ You could also choose to compress the jpg image and then encode it in base64.
+ Alternatively, on my own branch, I moved the save method of the raw image outside of this function and into the lidhandler.

I don't think we gain much more information from the undistorted picture. Better to send the raw image + the distortion matrix - both of which can be accessed in lidhandler.py